main.go

```diff
@@ -86,24 +86,36 @@ func main() {
 
 	for {
 		for _, u := range urls {
-			log.Printf("Backing up %s", u.Database)
-			file := newFileName(u.Database, u.DbType)
+			dumpFile := newFileName(u.Database, u.DbType)
 
-			if err = RunDump(&u, file); err != nil {
+			log.Printf("Backing up %s", u.Database)
+			if err = RunDump(&u, dumpFile); err != nil {
 				log.Printf("WARNING: Failed to dump database: %s", err)
 				continue
 			}
 
-			log.Printf("Uploading %s to %s", file, *s3Bucket)
+			log.Printf("Compressing %s", dumpFile)
```

Comment:
We might want to have this behind a command line flag, just because it doesn't make sense to spend the extra CPU time for really small dumps.

Reply:
Yeah sure I can add a flag for that. But IMHO the cpu time to compress very small dumps should be negligible (at least it would be for me).